### PR TITLE
chore: update docs

### PR DESCRIPTION
**Pull request by alanthai**

### CODE REVIEW

File: docs/Components/Card.md [hunk 1]
```diff
@@ -44,6 +57,6 @@ render(ExtensionPoint.MyExtension, () => <App />);
 
 | Name        | Type         | Description                                           | Required |
 | ----------- | ------------ | ----------------------------------------------------- | -------- |
-| content     | `string`     | Action label text.                                    | ☑️        |
+| content     | `string`     | Action label text.                                    | ☑️       |
```
Comment (hannachen) [inline docs/Components/Card.md:60]: Why is the space gone? 🤔

File: docs/Components/Modal.md [hunk 2]
```diff
@@ -19,32 +26,35 @@ render(ExtensionPoint.MyExtension, (root) => {
...
-| Name             | Type         | Description                         | Required |
-| ---------------- | ------------ | ----------------------------------- | -------- |
-| open             | `boolean`    | Whether the modal is open.         | ☑️        |
-| title            | `string`     | Title content for the modal.        | ☑️        |
-| primaryAction    | `Action`     | Modal's primary action.             |          |
-| secondaryActions | `Action[]`   | Modal's secondary action(s).        |          |
-| onClose          | `() => void` | Callback when the modal is closed.  | ☑️        |
+| Name             | Type         | Description                        | Required |
+| ---------------- | ------------ | ---------------------------------- | -------- |
+| open             | `boolean`    | Whether the modal is open .        | ☑️       |
```
Comment (hannachen) [inline docs/Components/Modal.md:48]: Extra whitespace?

File: docs/Components/Page.md [hunk 3]
```diff
@@ -19,27 +21,30 @@ render(ExtensionPoint.MyExtension, (root) => {
...
-| title            | `string`         | Page title, in large type.                                                                | ☑️       |
-| subTitle         | `string`         | Page subtitle, in regular type.                                                           |          |
-| primaryAction    | `Action`         | Primary page-level action. See [Action](#Action) action.                                  |          |
-| secondaryActions | `Action[]`       | Collection of secondary page-level actions. See [Action](#Action)actions.                 |          |
-| thumbnail        | `ThumbnailProps` | Thumbnail that precedes the title. See [Thumbnail Props API](./Thumbnail.md#Props%20API). |          |
+| Name             | Type             | Description                                                                              | Required |
+| ---------------- | ---------------- | ---------------------------------------------------------------------------------------- | -------- |
+| title            | `string`         | Page title, in large type                                                                | ☑️       |
```
Comment (hannachen) [inline docs/Components/Page.md:43]: Nit: do we want to be consistent about using period (`.`) in props descriptions?

File: docs/ExtensionPoints/ProductSubscription/README.md [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+
+**App overlay container**: A full-screen container designed for complex tasks, like configuring a subscription for the first time.
+
+#### Add, Remove (Modal container)
+
+| Name               | Type       | Description                                                                            | Required |
+| ------------------ | ---------- | -------------------------------------------------------------------------------------- | -------- |
+| close              | `function` | Closes the container and the extension                                                 |          |
+| done               | `function` | Notify Shopify Admin that the extension workflow is complete and data has been updated |          |
+| setPrimaryAction   | `function` | Sets the primary action content and callback when the action is clicked                |          |
```
Comment (hannachen) [inline docs/ExtensionPoints/ProductSubscription/README.md:109]: Is it set when clicked? Or when the extension is loaded/extension mode action is clicked?

File: docs/TESTING.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+## Testing Locally
+
+Run `shopify serve` to start the simulator, which runs your extension code locally. Navigate to the localhost url logged in your CLI to see the simulator. Here, you can see the changes you have made locally.
+
+### Adding mock data
+
+In the simulator, you can add mock data for each extension point.
+
+**Note:**  The shape of the data is different for each extension point.
```
Comment (hannachen) [inline docs/TESTING.md:11]:
```suggestion
**Note:** The shape of the data is different for each extension point.
```

File: docs/TESTING.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+## Testing on a Shop
+
+Once your Argo Extension is ready, push your code to Shopify and publish a version to see your extension in a Shopify store.
```
Comment (hannachen) [inline docs/TESTING.md:15]: @ow is `Argo Extension` ok here? Or should we reword to match public docs?

File: docs/Utilities/SessionToken.md [hunk 7]
```diff
@@ -46,11 +52,14 @@ function App() {
...
 ```
 
 ## Session Token API
 
-| Name            | Type                                | Description                | Required |
-| --------------- | ----------------------------------- | -------------------------- | -------- |
-| getSessionToken | `() => Promise<string | undefined>` | Get a fresh session token. | ☑️       |
+| Name            | Type                                 | Description                | Required |
+| --------------- | ------------------------------------ | -------------------------- | -------- |
+| getSessionToken | `() => Promise<string \| undefined>` | Get a fresh session token. | ☑️       |
```
Comment (hannachen) [inline docs/Utilities/SessionToken.md:65]:
```suggestion
| getSessionToken | `() => Promise<string | undefined>` | Get a fresh session token. | ☑️       |
```

**Review by hannachen — APPROVED** [review]
Looks great, thanks Alan! Just a few tiny edits.

File: docs/Components/Card.md (hunk 1 above)
Comment (henrytao-me) [reply docs/Components/Card.md:60]: VSCode markdown auto format  is weird. It does similar thing to me too :D

**Review by henrytao-me — APPROVED** [review]
LGTM

File: docs/Components/Page.md (hunk 3 above)
Comment (alanthai) [reply docs/Components/Page.md:43]: I have no idea xD

File: docs/Utilities/SessionToken.md (hunk 7 above)
Comment (alanthai) [reply docs/Utilities/SessionToken.md:65]: This back slash escapes the | or else it'll mistaken it for table cell separator on markdown.

File: docs/ExtensionPoints/ProductSubscription/README.md (hunk 4 above)
Comment (alanthai) [reply docs/ExtensionPoints/ProductSubscription/README.md:109]: Yeah I think you're right the sentence could be interpreted in 2 ways. Maybe "Sets the primary action content and action pressed callback". Something like that?

File: docs/TESTING.md (hunk 6 above)
Comment (alanthai) [reply docs/TESTING.md:15]: Maybe lose the Capital E?

File: docs/ExtensionPoints/ProductSubscription/README.md (hunk 4 above)
Comment (alanthai) [reply docs/ExtensionPoints/ProductSubscription/README.md:109]: Now that disabled is added to this, maybe we should simplify this to `Sets the primary action`, then add a table below for the actions prop.

File: docs/Utilities/SessionToken.md (hunk 7 above)
Comment (hannachen) [reply docs/Utilities/SessionToken.md:65]: I see...! I guess my IDE's .md preview doesn't have the same issue. Good to know.

File: docs/Components/Card.md (hunk 1 above)
Comment (hannachen) [reply docs/Components/Card.md:60]: Ooh I see.. didn't realize these came from auto format!

File: docs/LAUNCHING.md [hunk 8]
```diff
@@ -0,0 +1,7 @@
+# Launching your extension to production
+
+**Note:** Currently, you need to publish a version of your extension before running it on a Shopify store.
+
+Learn how to [test your extension on a Shopify store](./TESTING.md#testing-on-shop).
+
+To test changes without changing the live version of your extension, add it to a test version of your app and publish there. Once your changes are ready, copy the Argo code to the extension on your live app.
```
Comment (ow) [inline docs/LAUNCHING.md:7]: With this new flow this isn't really true, right? They can perform tests on their app extensions without affecting live. Should we not be specific about that? 

E.g. `Some app extensions, such as those that support Argo, now offer the ability to test changes in development stores without making changes to production or creating a copy of your app. Learn more about how to test your extensions.`

**Pull request merged**